src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -270,18 +271,33 @@ void DbWrapper::writeClockNetsToDb(Clock& clockNet) {
                                         inputPin = getFirstInput(sink);
                                 } else {
                                         inputPin = _block->findITerm(inst->getName().c_str());
```

Comment:
Why does it make sense to lookup an ITerm based on an inst name? Why are strings being used at all?

Reply:
The ClockInst object in TritonCTS represents both buffers and FFs. In the case of a FF, the name of the clock pin is saved instead. I've changed so that an ITerm reference is saved when this is the case.
Most of these (name lookups) are still here from old versions of TritonCTS. Since I'm experimenting with the clock tree builder algorithm right now, I'll make sure to clear any leftover name lookups on my next PR.